android/WALT/app/src/main/java/org/chromium/latency/walt/AudioTest.java

```diff
@@ -279,4 +311,18 @@ private void finishAndShowStats() {
             resultHandler.onResult(deltas);
         }
     }
+
+    private void finishRecordingTest() {
+        clockManager.checkDrift();
+
+        logger.log("deltas: " + deltas.toString());
+        logger.log(String.format(Locale.US,
+                "Median audio latency %.1f ms",
```

Comment:
Let's add "recording" to the phrasing here, to make it easy to distinguish from the input when reading the log.